crates/sncast/tests/docs_snippets/validation.rs

```diff
@@ -63,8 +46,8 @@ fn test_docs_snippets() {
         let exit_code = output.status.code().unwrap_or_default();
         let stderr = String::from_utf8_lossy(&output.stderr);
 
-        assert_valid_snippet(exit_code != 2, snippet, "sncast", &stderr);
+        assert_valid_snippet(exit_code != 2, snippet, &stderr);
```

Comment:
Why not check if it's `0`?

Reply:
As I mentioned in [this comment](https://github.com/foundry-rs/starknet-foundry/pull/2687#issuecomment-2490793232), sncast commands output validation will be added in the next PR. Currently, we simply check if clap didn't raise any errors, i.e. error code is different than 2.